packages/amplify-category-api/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.js

```diff
@@ -635,7 +635,7 @@ function newLambdaFunction(context, path) {
     },
   };
   return add(context, 'awscloudformation', 'Lambda', params).then(resourceName => {
-    context.print.success('Succesfully added the Lambda function locally');
+    context.print.success('Successfully added the Lambda function locally');
```

Comment:
double check this isn't going to break any e2e tests

Reply:
It is replaced everywhere I checked, did not find any of it.